main.go

```diff
@@ -28,6 +28,10 @@ func (e *errorString) Error() string {
 }
 
 func upsertPlayer(key string, new_p Player) error {
+	if new_p.Score < 10 || len(new_p.Code) <= 32 {
```

Comment:
`return [30,30]` is a valid program, so I think we should make this limit smaller